rootfs/docker_build.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/sh
+
```

Comment:
Can we add a check and a help message here asking for exactly one positional parameter?